### PR TITLE
Adding monitoring and other setup files for Percona and removing event router dependency.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
Signed-off-by: ishangupta-ds <ishan.gupta@mayadata.io>

## Proposed changes

* Adding Prometheus setup file changes for Percona (mysqld-exporter for non-PMM and configuration for enabling PMM).
* Adding operator, CRD, CR, secrets and RBAC files for Percona setup.
* Adding helm installation steps for PMM server and client. 
* Adding chaos interleaved PXC Galera Node summary dashboard (to be used with PMM)
* Adding chaos interleaved SQL overview dashboard (to be used with mysqld-exporter)
* Adding pod network loss chaos engine as a chaos injector (experiment) to utilities.
* Updating chaos exporter deployment file (bug fixes which were needed due to version update)
* Removing event router
* Updating chaos exporter files
* Restructuring the monitoring directory and README
* Updating Grafana dashboards
* Adding scrape jobs and service monitor for mysqld exporter, and scrape jobs for node and Kube state metrics exporter
* Adding cAdvisor scrape config
* Upgrading all images

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- None.

## Special notes for your reviewer:
Percona monitoring setup and related files.

### CODE REVIEW

File: monitoring/grafana-dashboards/percona-sql/MySQL-Overview-Interleaved.json [hunk 1]
```diff
@@ -0,0 +1,5208 @@
+{
```
Comment (ksatchit) [inline monitoring/grafana-dashboards/percona-sql/MySQL-Overview-Interleaved.json:1]: Do we really need these dashboards. I suppose the galera pxc ones are good enough?

File: monitoring/utils/metrics-exporters-with-service-monitors/litmus-metrics/chaos-exporter/chaos-exporter.yaml [hunk 2]
```diff
@@ -17,10 +17,9 @@ spec:
...
     spec:
       containers:
-      - image: litmuschaos/chaos-exporter:ci
-        imagePullPolicy: Always
-        name: chaos-exporter
-      serviceAccount: litmus
+        - image: litmuschaos/chaos-exporter:v1.13.x
+          imagePullPolicy: Always
+          name: chaos-exporter
       serviceAccountName: litmus
```
Comment (ksatchit) [inline monitoring/utils/metrics-exporters-with-service-monitors/litmus-metrics/chaos-exporter/chaos-exporter.yaml:23]: I see that with portal based installations this is litmus-cluster-scope. Do we need to change this or add a comment here?

File: monitoring/utils/metrics-exporters-with-service-monitors/mysqld-exporter/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
```
Comment (ksatchit) [inline monitoring/utils/metrics-exporters-with-service-monitors/mysqld-exporter/deployment.yaml:1]: Are we using this ?

File: monitoring/utils/sample-application-under-test/percona/cr.yaml [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+#  annotations:
+#    percona.com/issue-vault-token: "true"
+spec:
+  crVersion: 1.6.0
+  secretsName: my-cluster-secrets
+  vaultSecretName: keyring-secret-vault
+  sslSecretName: my-cluster-ssl
+  sslInternalSecretName: my-cluster-ssl-internal
+  logCollectorSecretName: my-log-collector-secrets
+  #  disableHookValidation: true
```
Comment (ksatchit) [inline monitoring/utils/sample-application-under-test/percona/cr.yaml:18]: I suppose we turned this On  prevent ssl errors. @uditgaurav ?

File: monitoring/grafana-dashboards/percona-sql/MySQL-Overview-Interleaved.json (hunk 1 above)
Comment (ishangupta-ds) [reply monitoring/grafana-dashboards/percona-sql/MySQL-Overview-Interleaved.json:1]: Yes @ksatchit we need this for a non PMM setup

File: monitoring/utils/metrics-exporters-with-service-monitors/litmus-metrics/chaos-exporter/chaos-exporter.yaml (hunk 2 above)
Comment (ishangupta-ds) [reply monitoring/utils/metrics-exporters-with-service-monitors/litmus-metrics/chaos-exporter/chaos-exporter.yaml:23]: we can add a comment for the scope but on the litmus-portal installation we would have for both scopes, this one is just for native monitoring setup

File: monitoring/utils/metrics-exporters-with-service-monitors/mysqld-exporter/deployment.yaml (hunk 3 above)
Comment (ishangupta-ds) [reply monitoring/utils/metrics-exporters-with-service-monitors/mysqld-exporter/deployment.yaml:1]: yes @ksatchit for non PMM method

File: monitoring/utils/sample-application-under-test/percona/cr.yaml (hunk 4 above)
Comment (ishangupta-ds) [reply monitoring/utils/sample-application-under-test/percona/cr.yaml:18]: should I switch this to on? On my setup I wasn't getting such error but we can disable it for greater compatibility @ksatchit  cc: @uditgaurav

**Review by ksatchit — APPROVED** [review]

**Comment by ksatchit** [comment]
@whitetiger1399 can you take a look too? Thanks

**Review by whitetiger1399 — APPROVED** [review]

**Pull request merged**